lua/telescope/previewers/term_previewer.lua

```diff
@@ -225,7 +225,8 @@ previewers.new_termopen_previewer = function(opts)
           return
         end
 
-        vim.fn.chansend(term_id, termcode)
+        vim.fn.chansend(term_id, termcode .. (utils.is_windows() and "\r\n" or ""))
```

Comment:
can you try this? the docs say when passing a list, we can include an final empty string to send a final newline. That might simplify things and remove the need to check for `is_windows`.
```suggestion
        vim.fn.chansend(term_id, { termcode, "" })
```

Reply:
If we do this, we will also affect unix-like systems with change, have you tested it preserves the same behavior on unix-like systems ? I will do the windows test next week

Reply:
Yeah my suggestion works on Linux. Just curious if it will on windows as well.